docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -124,6 +124,100 @@ elastic-agent-fj2z9             1/1     Running   0          81m
 elastic-agent-hs4pb             1/1     Running   0          81m
 ------------------------------------------------
 
+[discrete]
+== Red Hat OpenShift configuration
+
+If you are using Red Hat OpenShift, you need to specify additional settings in
+the manifest file and enable the container to run as privileged.
+
+. In the manifest file, modify the `agent-node-datastreams` ConfigMap and adjust inputs:
++
+--
+* `kubernetes-cluster-metrics` input:
+** If `https` is used to access `kube-state-metrics`, then to all `kubernetes.state_*` datasets should be added:
++
+[source,yaml]
+------------------------------------------------
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  ssl.certificate_authorities:
+    - /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+------------------------------------------------
+* `kubernetes-node-metrics` input:
+** `kubernetes.controllermanager` datastream condition should be change to:
++
+[source,yaml]
+------------------------------------------------
+condition: ${kubernetes.labels.app} == 'kube-controller-manager'
+------------------------------------------------
+** `kubernetes.scheduler` datastream condition should be changed to:
++
+[source,yaml]
+------------------------------------------------
+condition: ${kubernetes.labels.app} == 'openshift-kube-scheduler'
+------------------------------------------------
+** `kubernetes.proxy` datastream configutation should look like:
++
+[source,yaml]
+------------------------------------------------
+- data_stream:
+    dataset: kubernetes.proxy
+    type: metrics
+  metricsets:
+    - proxy
+  hosts:
+    - 'localhost:29101'
+  period: 10s
+------------------------------------------------
+** to all datastreams that connnect to `https://${env.NODE_NAME}:10250` should be added:
++
+[source,yaml]
+------------------------------------------------
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  ssl.certificate_authorities:
+    - /path/to/ca-bundle.crt
+------------------------------------------------
+NOTE: `ca-bundle.crt` can be any CA bundle that contains the issuer of the certificate used in the Kubelet API.
+According to each specific installation of Openshift this can be found either in `secrets` or in `configmaps`.
+In some installations it can be available as part of the service account secret, in
+`/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`.
+In case of using https://github.com/openshift/installer/blob/master/docs/user/gcp/install.md[Openshift installer]
+for GCP then the following `configmap` can be mounted in Metricbeat Pod and use `ca-bundle.crt`
```

Comment:
Did you mean elastic-agent pod?

Reply:
thank you for pointing it out! done - https://github.com/elastic/observability-docs/pull/1498/commits/eca949819181efb5fff36f948943d03b5f389b87